# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bbdock/hotfix-string.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- bbdock-0.2.9/src/Dock.cc.vanilla	2022-03-31 14:52:57.505225300 +0200
+++ bbdock-0.2.9/src/Dock.cc	2022-03-31 14:53:32.530227106 +0200
@@ -31,6 +31,8 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <unistd.h>
+#include <string.h>
+
 #include <iostream>
 #include <string>
 
